info/v1/container.go

```diff
@@ -395,10 +395,16 @@ type MemoryStats struct {
 
 	Failcnt uint64 `json:"failcnt"`
 
+	CpuSet CPUSetStats `json:"cpuset,omitempty"`
```

Comment:
curious to get your thoughts: does this make sense under `MemoryStats`? 

Since this is taken from the cpuset controller, maybe this makes sense to add a new `CpuSets` top level field under `ContainerStats` which will contain the relevant metrics from cpuset: https://man7.org/linux/man-pages/man7/cpuset.7.html

As I understand currently all the fields under `MemoryStats` actually come from the memory cgroup controller stats. 

Thoughts @kwisniewski98 @Creatone @iwankgb ?

Reply:
That will be a good place for other CPUSet metrics.

Reply:
Done